tests/abc_atlas_cache/utils.py

```diff
@@ -0,0 +1,184 @@
+from typing import Optional
+import boto3
+import hashlib
+from moto import mock_aws
+import unittest
+import tempfile
+from pathlib import Path
+from abc_atlas_access.abc_atlas_cache.file_attributes import \
+    CacheFileAttributes
+
+
+def create_bucket(region_name: str,
```

Comment:
I'm being a little paranoid here, but can we make this a member of BaseCacheTestCase so no one accidentally calls this outside of a `@mock_aws` invocation.

Or else wrap this function in `@mock_aws` to be doubly-sure. When I first saw this function I did a double-take before realizing that it is only ever invoked in classes wrapped with `@mock_aws`